### PR TITLE
chore(dataproc): release 0.6.1

### PR DESCRIPTION
**Pull request by codyoss**
This pull request was generated using releasetool.

### CODE REVIEW

**Comment by codyoss** [comment]
@busunkim96 I think was supposed to add a label to the PR(via releasetool), but I seem to not have the proper access so the cli flacked out on me. Seems to have created the PR though. What label should I manually add to keep the process going?

File: dataproc/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,22 @@
 
 [1]: https://pypi.org/project/google-cloud-dataproc/#history
 
+## 0.6.1
+
+11-12-2019 08:24 PST
+
+- docs: add python 2 sunset banner to documentation ([#9036](https://github.com/googleapis/google-cloud-python/pull/9036))
```
Comment (tseaver) [inline dataproc/CHANGELOG.md]: Each changelog entry needs to be moved to the correct section below (`### Documentation` in this case).

Also, please strip the prefixes, capitalize, and add a period.  In the future, those changes will be applied by `release-please`, but it won't work until we split up the repository.

```suggestion
- Add python 2 sunset banner to documentation. ([#9036](https://github.com/googleapis/google-cloud-python/pull/9036))
```

File: dataproc/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,22 @@
...
+
+### Implementation Changes
```
Comment (tseaver) [inline dataproc/CHANGELOG.md]: After moving the generated entries into the appropriate sections, please remove any empty sections.

**Comment by busunkim96** [comment]
@codyoss You can add the label `autorelease: pending` manually (once you have write access to the repo)

**Comment by busunkim96** [comment]
You're good to merge this now @codyoss 👍

**Pull request merged**